controller/api/serializers.py

```diff
@@ -239,23 +239,22 @@ def validate_domain(self, value):
         """
         Check that the hostname is valid
         """
-        match = re.match(
-            r'^(\*\.)?(' + settings.APP_URL_REGEX + r'\.)*([a-z0-9-]+)\.([a-z0-9]{2,})$',
-            value)
-        if not match:
+        if len(value) > 255:
```

Comment:
I believe a Validator already exists for this: https://github.com/django/django/blob/master/django/core/validators.py#L294

There's some pretty useful validators at `django.core.validators`. We should use them more often :)